docs/HIV_cohort_selection.md

```diff
@@ -0,0 +1,78 @@
+# HIV Cohort Selection App
+### Try it out at https://aids.niaiddata.org/analysis/ndhHIV
+------------------------
+
+The HIV Cohort Selection App enables researchers to query the HIV infection controller types (Long-Term Non-Progressors, Post-Treatment Controllers and Elite Controllers) based on the customized rate of HIV progression.
+
+
+Example of HIV Cohort Selection App:
+![HIV_Cohort_Selection example](cohort_selection_example.png)
+## Background
+The human immunodeficiency virus (HIV) infection is a dynamic process and has a variable rate of progression in infected people. Despite the varying disease progression rates, the majority of HIV-infected individuals eventually progress to AIDS. Interestingly, a small group of people who are HIV positive remain healthy for long periods of time without medications. These people, known as Long-Term Non-Progressors, Post-Treatment Controllers and Elite Controllers), have properties within their immune systems that control the virus. They can help researchers determine how their immune systems control HIV disease. This knowledge is critical for developing new treatments and vaccines. 
```

Comment:
You have an open parenthesis at the end of:

Elite Controllers)

Reply:
good catch!